### PR TITLE
DevTools Tips 15 > Different ways to open DevTools

### LINKED ISSUE #3975: [devtools-tips] 15 - Top 7 ways to open DevTools
Details: b/233724275
Deadline: Oct 26
Notes: Do not merge the PR when ready, tag @jecfish to merge in time with video publish

Also refresh relevant docs, and include the video
- https://developer.chrome.com/docs/devtools/open/
- https://developer.chrome.com/docs/devtools/overview/

### PR DESCRIPTION
**Pull request by sofiayem**
Promo post for DTT15. Fixes #3975.

Also, refreshed the "Open Chrome DevTools" doc.

Publish on Oct 26.

### CODE REVIEW

File: site/en/blog/devtools-tips-15/index.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+    <td><kbd>Fn</kbd> + <kbd>F12</kbd></br><kbd>Cmd</kbd> + <kbd>Option</kbd> + <strong>I</strong></td>
+  </tr>
+</tbody>
+</table>
+
+The easy way to memorize the shortcuts:
+
+- **C** stands for CSS
+- **J** for JavaScript
+- **I** designates your choice
```
Comment (rachelandrew) [inline site/en/blog/devtools-tips-15/index.md:67]:
```suggestion
- **C** stands for CSS.
- **J** for JavaScript.
- **I** designates your choice.
```

File: site/en/docs/devtools/open/index.md [hunk 2]
```diff
@@ -4,68 +4,125 @@ title: "Open Chrome DevTools"
...
+    <td><kbd>Fn</kbd> + <kbd>F12</kbd></br><kbd>Cmd</kbd> + <kbd>Option</kbd> + <strong>I</strong></td>
+  </tr>
+</tbody>
+</table>
+
+Here's an easy way to memorize the shortcuts:
+
+- **C** stands for CSS
+- **J** for JavaScript
+- **I** designates your choice
```
Comment (rachelandrew) [inline site/en/docs/devtools/open/index.md:79]:
```suggestion
- **C** stands for CSS.
- **J** for JavaScript.
- **I** designates your choice.
```

**Review by rachelandrew — APPROVED** [review]

File: site/en/docs/devtools/open/index.md [hunk 3]
```diff
@@ -4,68 +4,125 @@ title: "Open Chrome DevTools"
...
+DevTools will automatically open for every new tab until you close Chrome.
 
 ## What's next?
 
-Congratulations! You've successfully unlocked the power of Chrome DevTools. Next, watch the following video to learn some useful shortcuts and settings for quicker DevTools navigation.
+{% Aside 'success' %}
+Congratulations! You've successfully unlocked the power of Chrome DevTools.
+{% endAside %}
+
+Next, watch the following video to learn some useful shortcuts and settings for quicker DevTools navigation.
```
Comment (jecfish) [inline site/en/docs/devtools/open/index.md:126]: Can we add a few more devtools docs links to help users learning devtools? e.g. goo.gle/devtools-customize.

File: site/en/docs/devtools/open/index.md [hunk 4]
```diff
@@ -4,68 +4,125 @@ title: "Open Chrome DevTools"
...
+</table>
+
+Here's an easy way to memorize the shortcuts:
+
+- **C** stands for CSS.
+- **J** for JavaScript.
+- **I** designates your choice.
+
+The **C** shortcut opens the **Elements** panel in {% Img src="image/NJdAV9UgKuN8AhoaPBquL7giZQo1/7s3JQLXmIQmQa4CFXaNv.png", alt="Inspect.", width="21", height="20" %} inspector mode. This mode shows you helpful tooltips when you hover over elements on a page.
+
```
Comment (jecfish) [inline site/en/docs/devtools/open/index.md:82]: Add a line to tell users that they can click on any elements to inspect its styles in the Styles pane.

**Review by jecfish — APPROVED** [review]
LGTM % comments. Thanks Sofia!

**Pull request merged**